### DISCUSSION TITLE
h3_get_pentagons()

### DISCUSSION THREAD

**Discussion by infoplane** [bytesandbrains/h3-pg#146] [General]
Hi, I'm playing around with the h3 extension for Postgres. Using the function h3_get_pentagons() is only returning 12 results irrespective of the resolution I set in the function. My understanding is that a resolution of zero should return 122 indexes which should increase with greater resolution. I was wondering if I have missed something.

I'm using h3 version 4.1.3, postgis version 3.2.5, 

Marcus

**Comment by zachasme** [comment]
Hi Marcus!

The number of hexagons will increase with greater resolution, but [there are always exactly 12 pentagons at every resolution](https://h3geo.org/docs/core-library/overview).

For questions regarding the H3 API itself you should refer to [the H3 core repository](https://github.com/uber/h3/discussions), this project simply wraps the functions in a PostgreSQL API.

Cheers,
Zach

**Reply by infoplane** [reply]
Hi Zach,

Many thanks for the clarification. I will have a look at the H3 core repository.

Cheers,

Marcus
